src/Uno.UWP/Globalization/NumberFormatting/INumberRounder.cs

```diff
@@ -0,0 +1,12 @@
+namespace Windows.Globalization.NumberFormatting
+{
+	public partial interface INumberRounder 
+	{
+		int RoundInt32( int value);
+		uint RoundUInt32( uint value);
+		long RoundInt64( long value);
+		ulong RoundUInt64( ulong value);
+		float RoundSingle( float value);
+		double RoundDouble( double value);
```

Comment:
nit: Keep formatting clean in non-generated file.

```suggestion
		int RoundInt32(int value);
		uint RoundUInt32(uint value);
		long RoundInt64(long value);
		ulong RoundUInt64(ulong value);
		float RoundSingle(float value);
		double RoundDouble(double value);
```